docs/tutorial.py

```diff
@@ -11,11 +11,11 @@
 
 Most newer devices require your TP-Link cloud username and password, but this can be omitted for older devices.
 
->>> from kasa import Device, Discover, Credentials
+>>> from kasa import Device, Discover
```

Comment:
```suggestion
>>> from kasa import Discover
```
I don't see `Device` being used here at all.

Reply:
Removed